Source/Basic Shapes/SvgPolyline.cs

```diff
@@ -15,7 +15,7 @@ public partial class SvgPolyline : SvgPolygon
 
         public override GraphicsPath Path(ISvgRenderer renderer)
         {
-            if ((_Path == null || this.IsPathDirty) && base.StrokeWidth > 0)

```

Comment:
Here also has the same condition.
https://github.com/svg-net/SVG/blob/c4e83167c6adfb8d6770eb109d1a03dd656096c2/Source/Basic%20Shapes/SvgLine.cs#L90

Probably don't need this either.

Reply:
This is a line that cannot be filled, so this may avoid calling unnecessary code.